dockerfiles/Dockerfile.tests

```diff
@@ -0,0 +1,174 @@
+WORKDIR /alb-sdk/python/avi/migrationtools/nsxt_converter/
+RUN python3 install_nsx_dependencies.py
+
+# Install terraform
+WORKDIR /
+RUN curl https://releases.hashicorp.com/terraform/${tf_version}/terraform_${tf_version}_linux_amd64.zip -o terraform_${tf_version}_linux_amd64.zip &&  \
+    unzip terraform_${tf_version}_linux_amd64.zip -d /usr/local/bin && \
+    rm -rf terraform_${tf_version}_linux_amd64.zip
+
+# Clone avitools and devops repositories
+RUN cd $HOME && \
+    git clone https://github.com/avinetworks/devops && \
+    git clone https://github.com/vmware/nsx-advanced-load-balancer-tools avitools && \
+    mkdir -p /avi/scripts && \
+    cp -r avitools/scripts/* /avi/scripts && \
+    rm -rf $HOME/avitools
+
+# Install go for AVI terraform provider
+RUN curl -L https://go.dev/dl/go${golang_version}.linux-amd64.tar.gz -o go${golang_version}.linux-amd64.tar.gz && \
+    rm -rf /usr/local/go && tar -C /usr/local -xzf go${golang_version}.linux-amd64.tar.gz && \
+    go version && \
+    rm go${golang_version}.linux-amd64.tar.gz
+
+# Clone terraform provider repository and build provider locally.
+RUN cd $HOME && \
+    git clone https://github.com/vmware/terraform-provider-avi && \
+    cd ~/terraform-provider-avi && \
+    git checkout ${branch} && \
+    make fmt . && \
+    go mod tidy && \
+    make build13 && \
+    mkdir -p $HOME/.terraform.d/ && \
+    mkdir -p $HOME/.terraform.d/plugin-cache && \
+    cd ~/terraform-provider-avi/examples/aws/cluster_stages/1_aws_resources && \
+    terraform init
+
+# Clone ansible repo and install ansible collections.
+RUN cd $HOME && \
+    git clone https://github.com/vmware/ansible-collection-alb && \
+    cd ~/ansible-collection-alb && \
+    git checkout ${branch} && \
+    ansible-galaxy collection build && \
+    ansible-galaxy collection install vmware-alb-*.tar.gz && \
+    pip3 install -r ~/.ansible/collections/ansible_collections/vmware/alb/requirements.txt
+
+# Verify all converters files.
+RUN touch list && \
+    echo '#!/bin/bash' > avitools-list && \
+    echo "echo "f5_converter.py"" >> avitools-list && \
+    echo "echo "netscaler_converter.py"" >> avitools-list && \
+    echo "echo "gss_convertor.py"" >> avitools-list && \
+    echo "echo "f5_discovery.py"" >> avitools-list && \
+    echo "echo "avi_config_to_ansible.py"" >> avitools-list && \
+    echo "echo "ace_converter.py"" >> avitools-list && \
+    echo "echo "virtualservice_examples_api.py"" >> avitools-list && \
+    echo "echo "config_patch.py"" >> avitools-list && \
+    echo "echo "vs_filter.py"" >> avitools-list && \
+    echo "echo "nsxt_converter.py"" >> avitools-list
+
+# Verify all script in avitools-list
+RUN for script in $(ls /avi/scripts); do echo "echo $script" >> avitools-list; done;
+
+# make executables
+RUN chmod +x avitools-list && \
+    cp avitools-list /usr/local/bin/ && \
+    echo "alias avitools-list=/usr/local/bin/avitools-list" >> ~/.bashrc
+
+# Clean out the cache
```

Comment:
Does this takes care of cleaning the cache for the ansible and terraform building we are doing here?

Reply:
Added changes to cleanup the cache for terraform.